supervisor/backups/manager.py

```diff
@@ -359,7 +359,7 @@ async def import_backup(
             return None
 
         # Load new backup
-        backup = Backup(self.coresys, tar_origin, backup.slug, None, backup.data)
+        backup = Backup(self.coresys, tar_origin, backup.slug, location, backup.data)
```

Comment:
Is this addressing a different problem than the one in the PR description?

Reply:
That is mentioned in the PR description too:

> Additionally the new test case for uploading a backup that already exists identified that upload backup was setting the location wrong so fix that as well.